security/pkg/util/jwtutil.go

```diff
@@ -49,6 +49,35 @@ func IsJwtExpired(token string, now time.Time) (bool, error) {
 
 }
 
+// GetAud returns the claim `aud` from the token. Returns nil if not found.
+func GetAud(token string) ([]string, error) {
+	claims, err := parseJwtClaims(token)
+	if err != nil {
+		return nil, err
+	}
+
+	if claims["aud"] == nil {
+		return nil, fmt.Errorf("no aud in the token claims")
+	}
+
+	switch aud := claims["aud"].(type) {
+	case []interface{}:
```

Comment:
instead of hand parsing should we have a struct we marshal this into?

Reply:
the `aud` claim could be either a single string or a list strings (`[]string`), I updated to use json.Marshal, should make the code a bit more readable.